src/core/chart-generators.js

```diff
@@ -425,13 +427,23 @@ function shortestUniqueTimeseriesNamingFunction(metadata, data) {
   if (variation.length === 0) {
     throw new Error('Error: cannot graph identical timeseries');
   }
+  
+  // Build a dictionary with the base name for each variable (typically
+  // either "Mean" or "Mean Count") from the variable config file. 
+  // Defaults to "Mean" since all data displayed by this graph is MYMs.
+  const variables = _.uniq(_.pluck(metadata, "variable_id"));
+  function getVarBasename(v) {
+    const fromConfig = getVariableOptions(v, 'seriesLegendString');
+    return _.isUndefined(fromConfig) ? "Mean" : fromConfig;
+  }
+  const basenameByVariable = _.object(variables, _.map(variables, getVarBasename));
```

Comment:
Similarly, `_.object` has to be replaced by something different, this time a compound of Lodash `_.fromPairs` and `_.zipObject`. (See [Migrating](https://github.com/lodash/lodash/wiki/Migrating).) Again, timing.